funds-manager/funds-manager-server/src/handlers.rs

```diff
@@ -89,6 +92,27 @@ pub(crate) async fn quoter_withdraw_handler(
     withdraw_request: WithdrawFundsRequest,
     server: Arc<Server>,
 ) -> Result<Json, warp::Rejection> {
+    // Get the price of the token
+    let maybe_price = server
+        .relayer_client
+        .get_binance_price(&withdraw_request.mint)
+        .await
+        .map_err(|e| warp::reject::custom(ApiError::InternalError(e.to_string())))?;
+
+    if let Some(price) = maybe_price {
```

Comment:
can we log a warning or error here if there's no price? want to make sure that this doesn't go unnoticed